classes/GlobalDirtEffect.sc

```diff
@@ -23,11 +23,13 @@ GlobalDirtEffect {
 
 	play { |group, outBus, dryBus, effectBus, orbitIndex|
 		this.release;
-		synth = Synth.newPaused(name.asString ++ numChannels,
-			[\outBus, outBus, \dryBus, dryBus, \effectBus, effectBus, \orbitIndex, orbitIndex] ++ state.asPairs,
-			group,
-			\addAfter
-		)
+		if(active) {
```

Comment:
actually, this may not be needed. It is ok, to have a paused synth in there. I'll fix this.